frontend/src/content/news.json

```diff
@@ -1,9 +1,14 @@
 {
   "2022": [
+    {
+      "date": "2022-04-04",
+      "title": "Metabolic Atlas 2.5 improves the about pages and fixes bugs",
```

Comment:
In news text we mention `<i>About</i> pages`capitalized and in italics, so it is unexpected to called them here `about pages`.

Reply:
Thanks and sorry that I missed this before re-requesting reviews. Changed to `About` in https://github.com/MetabolicAtlas/MetabolicAtlas/pull/942/commits/b4d86757289113671bd36523fdd3dfa43245e83c now. The title currently doesn't parse any html tags. Do you want us to allow that so we can make it italic as well?

Reply:
For consistency, I think it would make sense, not sure if we need a separate issue.

Reply:
Thanks for the quick response! I felt like it was relevant and small enough to be included so I added it here: https://github.com/MetabolicAtlas/MetabolicAtlas/pull/942/commits/75cab8f594b17aafdc2a238b2096870004f9e364

Reply:
Awesome!